refinery/user_files_manager/tests.py

```diff
@@ -0,0 +1,16 @@
+import logging
+
+from django.test import TestCase
+
+
+logger = logging.getLogger(__name__)
+
+
+class UserFilesAPITests(TestCase):
+    pass
+    # TODO: Fails, because solr isn't running.
+    # Should I mock it, or make this an integration test?
+
+    # def test_get(self):
+    #     response = self.client.get('/api/v2/user/files/')
+    #     self.assertEqual(response.status_code, 200)
```

Comment:
That doesn't seem to be one that hits solr, which is the problem I'm dealing with.